File: shared/rules/cors-allow-all.py
import os

# ruleid: bugledger-cors-allow-all
allow_origins = "*"

# ruleid: bugledger-cors-allow-all
allow_origins = ["*"]

# ruleid: bugledger-cors-allow-all
cors_origins = "*"

# ruleid: bugledger-cors-allow-all
CORS_ALLOW_ALL_ORIGINS = True

# ok: bugledger-cors-allow-all
allow_origins = "https://myapp.com"

# ok: bugledger-cors-allow-all
allow_origins = ["https://myapp.com", "https://admin.myapp.com"]

# ok: bugledger-cors-allow-all
cors_origins = os.environ["ALLOWED_ORIGINS"]

File: shared/rules/cors-allow-all.js
const cors = require("cors");

// ruleid: bugledger-cors-allow-all
app.use(cors({ origin: "*" }));

// ruleid: bugledger-cors-allow-all
res.setHeader("Access-Control-Allow-Origin", "*");

// ok: bugledger-cors-allow-all
app.use(cors({ origin: "https://myapp.com" }));

// ok: bugledger-cors-allow-all
res.setHeader("Access-Control-Allow-Origin", "https://myapp.com");
